xbmc/games/addons/GameClientMouse.h

```diff
@@ -21,6 +21,8 @@
 
 #include "input/mouse/IMouseInputHandler.h"
 
+#define MOUSE_CONTROLLER_ID "game.controller.mouse"
```

Comment:
Is there any way we can avoid defining this here and use the value from here? https://github.com/garbear/xbmc/blob/retroplayer-mouse/xbmc/input/mouse/MouseWindowingButtonMap.cpp#L28